### ISSUE TITLE
If I specifically ask for one of the suggestions to be added to my list it should be automatically selected

### ISSUE THREAD

**Issue by brodonn** [n3xah/issues#9]
I asked for a list of book recommendations and then specified which book should be added to the list. I think if you specifically say "add this book to my list" then it should be automatically selected so you don't need to then select and confirm.

**Comment by brentbahry** [comment]
Yeah I think this is a good point regarding the suggestion flow. We defaulted a while back to you needing to manually confirm any suggestions you're promoting to be thoughts because the assistant makes mistakes sometimes, and we want to prevent data loss at all costs. However, suggestions to create new things wouldn't result in data loss and maybe we can make that experience more simple like how you're mentioning.

In general we're definitely planning to improve the suggestion flow because it's one of the core features of the app at present and needs to require as little effort from you as possible.

In the future we will also likely implement "undo" functionality and that could enable us to remove a lot of the suggestion approval process if "ctrl+z" can undo any mistakes the assistant makes.
